addons/website/static/src/xml/website.editor.xml

```diff
@@ -14,6 +14,12 @@
         </label>
     </t>
 
+    <t t-extend="wysiwyg.widgets.link">
+        <t t-jquery="#o_link_dialog_url_input" t-operation="after">
+            <small class="form-text text-muted">Hint: Type '/' to search an existing page and '#' to link to an anchor.</small>
```

Comment:
I see this was already in the `linkTools` template (though worded slightly differently). Is this so the hint appears in the dialog as well because it was missing somehow?

Reply:
Yes, without this commit, the hint is only on the sidebar but not the modal.